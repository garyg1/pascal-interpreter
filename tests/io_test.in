5
-12.3E12
