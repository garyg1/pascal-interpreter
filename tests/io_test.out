5.0
-1.23E13
